Rudder-Appsflyer/Classes/RudderAppsflyerIntegration.m

```diff
@@ -96,7 +98,7 @@ - (void) processRudderEvent: (nonnull RSMessage *) message {
             } else if ([eventName isEqualToString:ECommCheckoutStarted]){
                 afEventName = AFEventInitiatedCheckout;
                 [self addCheckoutProperties:properties params:afProperties];
-            } else if ([eventName isEqualToString:ECommOrderCompleted]) {
+            } else if ([eventName isEqualToString:FIRSTPURCHASE]) {
```

Comment:
Why `ECommOrderCompleted` deleted?

Reply:
Added it back now, it shouldn't be deleted.